gradle.properties

```diff
@@ -25,7 +25,7 @@ JUNIT_VINTAGE_VERSION=5.5.2
 KINDEDJ_VERSION=1.1.0
 KOTEST_VERSION=4.1.0
 KOTLIN_METADATA_VERSION=1.4.0
-KOTLIN_TEST_VERSION=3.3.3
+KOTLIN_TEST_VERSION=4.1.1
```

Comment:
Thanks @aballano !! `KOTEST_VERSION` already exists because it's being used for `arrow-fx-coroutines` so far.

Reply:
Thanks! I'll change it 🤦